### PR TITLE
facturial done

### PR DESCRIPTION
**Pull request by DavidNogueira**

### CODE REVIEW

File: factorial.js [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /* 1. Write a recursive JavaScript function to calculate the factorial of a number. The factorial of a non-negative integer n, denoted by n!, 
 is the product of all positive integers less than or equal to n. For example, 5! = 5 x 4 x 3 x 2 x 1 = 120  */
 
+function factorial(x) {
+    if (x === 0) {
+        return 1;
+    }
+    return x * factorial(x - 1);
+}
+console.log(factorial(5));
```
Comment (brunodevesa) [inline factorial.js]: well done. But before merge, please transform the function using the ternary operator, in order to simplify the code

Comment (DavidNogueira) [reply factorial.js]: Got it

File: factorial.js [hunk 2]
```diff
@@ -2,9 +2,6 @@
 is the product of all positive integers less than or equal to n. For example, 5! = 5 x 4 x 3 x 2 x 1 = 120  */
 
 function factorial(x) {
-    if (x === 0) {
-        return 1;
-    }
-    return x * factorial(x - 1);
+   return x == 0 ? (1) : (x * factorial(x - 1))
```
Comment (brunodevesa) [inline factorial.js]: please clarify the use of == operator instead of the === operator.

Comment (DavidNogueira) [reply factorial.js]:
```suggestion
   return x === 0 ? (1) : (x * factorial(x - 1))
```

**Pull request merged**